learing_journal/tests.py

```diff
@@ -103,21 +104,34 @@ def test_lists_body(new_session):
 def test_lists_date(new_session):
     '''tests wether list() pull out correct data from db'''
     from .views.default import home
-    new_session.add(MyModel(title='test1', body='test2', date='test3'))
+    new_session.add(Journal(title='test1', body='test2',
+                    date=datetime.datetime.strptime('August 23, 2016', '%B %d, %Y')))
     new_session.flush()
     result = home(dummy_http_request(new_session))
     for entry in result['entries']:
-        assert entry.date == 'test3'
+        assert entry.date.strftime('%B %d, %Y') == 'August 23, 2016'
 
 
-def test_create(new_session):
+def test_create_get(new_session):
     """
-    Tests whether create() returns  {'poject': 'learning_journal'}
+    Tests whether create() returns  {'title': '', 'body': '', 'error': ''}
     on 'GET' request.
     """
     from .views.default import create
     assert create(dummy_http_request(new_session)) == \
-        {'poject': 'learning_journal'}
+        {'title': '', 'body': '', 'error': ''}
+
+# causes: ../lib/python3.5/site-packages/zope/interface/registry.py:172: ComponentLookupError
+# def test_create_post(new_session):
+#     """
+#     Tests whether create() returns  HTTPFound on 'POST' request.
+#     """
+#     from .views.default import create
+#     # new_session.add(Journal(title='test1', body='test2',
+#     #                 date=datetime.datetime.strptime('August 23, 2016', '%B %d, %Y')))
+#     new_session.flush()
+#     with pytest.raises(HTTPFound):
+#         create(dummy_http_request_post('test1', 'test2', datetime.datetime.strptime('August 23, 2016', '%B %d, %Y'), new_session))
```

Comment:
Your create view returns an `HTTPFound` when you create a new post. One of the issues though is that in the test app you set up above, you don't include your "routes" like you did in your `__init__.py` file. As such, the test app that you're using to test your functionality has no awareness of how your routes connect to each other.